packages/core/src/common/predicates.spec.ts

```diff
@@ -148,45 +123,29 @@ describe('isNumber', () => {
   });
 });
 
-describe('isMinimalStaticViewPort', () => {
-  it('returns false when the viewport is minimal live viewport config', () => {
-    const viewport: MinimalLiveViewport = {
-      yMin: 0,
-      yMax: 10,
+describe('isHistoricalViewport', () => {
+  it('returns false when the viewport is a duration viewport', () => {
+    const viewport: DurationViewport = {
       duration: 1000,
     };
 
-    expect(isMinimalStaticViewport(viewport)).toBeFalse();
+    expect(isHistoricalViewport(viewport)).toBeFalse();
   });
 
-  it('returns true when the viewport is minimal static viewport config', () => {
-    const viewport: MinimalStaticViewport = {
-      yMin: 0,
-      yMax: 10,
+  it('returns true when the viewport is a historical viewport', () => {
+    const viewport: HistoricalViewport = {
       start: new Date(),
       end: new Date(),
     };
 
-    expect(isMinimalStaticViewport(viewport)).toBeTrue();
+    expect(isHistoricalViewport(viewport)).toBeTrue();
   });
 
   it('returns false when the end date is missing', () => {
-    const viewport: Omit<MinimalStaticViewport, 'end'> = {
-      yMin: 0,
-      yMax: 10,
-      start: new Date(),
-    };
-
-    expect(isMinimalStaticViewport(viewport as unknown as MinimalLiveViewport)).toBeFalse();
+    expect(isHistoricalViewport({ start: new Date() } as unknown as DurationViewport)).toBeFalse();
   });
 
   it('returns false when the start date is missing', () => {
-    const viewport: Omit<MinimalStaticViewport, 'start'> = {
-      yMin: 0,
-      yMax: 10,
-      end: new Date(),
-    };
-
-    expect(isMinimalStaticViewport(viewport as unknown as MinimalLiveViewport)).toBeFalse();
+    expect(isHistoricalViewport({ end: new Date() } as unknown as DurationViewport)).toBeFalse();
```

Comment:
Why are these object literals cast as unknown? Looks like a hack?

Reply:
this is necessary because i'm passing invalid inputs into the utility function that don't adhere to the typescript types but open to other suggestions